### PR TITLE
General overview of how the tool works

### PR DESCRIPTION
**Pull request by rhcarvalho**
Explain with diagrams and words why, how and what is happening.
Remove stale manual steps.

### CODE REVIEW

**Comment by rhcarvalho** [comment]
@AbhiPrasad this is probably easier to review using the "Split" or "Rich" views.

File: README.md [hunk 1]
```diff
@@ -4,73 +4,88 @@ This repository contains reproducible benchmarks to evaluate the instrumentation
...
+       └───────────────────────────────┘
 ```
 
-## How It Works
-
-The `sentry-sdk-benchmark` command automates the following steps:
+We provide a [collection of prepackaged web apps](platform/) implemented in different programming languages and using different frameworks.
 
-1. Start `baseline` app:
+For each original app in a "baseline" directory, there is a slightly modified app in a corresponding "instrumented" directory which is a copy of the original app with minimal changes to add Sentry instrumentation (error and performance monitoring).
```
Comment (AbhiPrasad) [inline README.md:29]:
```suggestion
For each original app in a "baseline" directory, there is a slightly modified app in a corresponding "instrumented" directory which is a copy of the original app with minimal changes to add Sentry instrumentation (error and performance monitoring). The "baseline" apps are implementations from [TechEmpower Framework Benchmarks](https://github.com/TechEmpower/FrameworkBenchmarks).
```

File: README.md [hunk 2]
```diff
@@ -4,73 +4,88 @@ This repository contains reproducible benchmarks to evaluate the instrumentation
...
+   │        │ │              │ │ and Test Orchestrator │ │   (fakerelay)    │
+   │        │ │              │ │      (loadgen)        │ │                  │
+   │        │ │              │ │                       │ │                  │
+   └────────┘ └──────────────┘ └───────────────────────┘ └──────────────────┘
+                                                          * only for
+                                                            instrumented apps
+```
 
-    You may repeat the command above a couple of times, observing that after the first run the maximum latency reported should drop down.
+Every app under test is supposed to interact with the database in response to requests from the load generator as described in the [Database Updates test](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#database-updates).
```
Comment (AbhiPrasad) [inline README.md:54]:
```suggestion
Every app under test is supposed to interact with a postgres database in response to requests from the load generator as described in the [Database Updates test](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#database-updates).
```

**Review by AbhiPrasad — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rhcarvalho) [reply README.md:29]: Your feedback makes me think we have a blurry line right now between implementation detail and general design.

I thought of this section of the main `README.md` as a high level overview to help understand what the project is about. And in #27, in `platform/README.md`, detail the current implementation constraints.

Let's have a quick chat to see if we can delineate the separation better.

Comment (rhcarvalho) [reply README.md:29]: We decided to embrace the current implementation constraints and leave generalizations for the future.

File: README.md (hunk 2 above)
Comment (rhcarvalho) [reply README.md:54]: 👍 

Will use "PostgreSQL" throughout for consistency.

```suggestion
Every app under test is supposed to interact with a PostgreSQL database in response to requests from the load generator as described in the [Database Updates test](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#database-updates).
```

**Pull request merged**